package errs

import "syscall"

var DIR_EXISTS_ERRNO syscall.Errno = syscall.ERROR_ALREADY_EXISTS
